src/mlagility/analysis/util.py

```diff
@@ -91,11 +91,21 @@ def populate_onnx_model_info(onnx_model) -> Dict:
         {
             "ir_version": getattr(model, "ir_version", None),
             "opset": getattr(model.opset_import[0], "version", None),
-            "size on disk (KiB)": round(
-                model.SerializeToString().__sizeof__() / 1024, 4
-            ),
         }
     )
+    try:
+        result_dict.update(
+            {
+                "size on disk (KiB)": round(
+                    model.SerializeToString().__sizeof__() / 1024, 4
+                ),
+            }
+        )
+    except ValueError:
+        # Models >2GB on disk cannot have their model size measured this
+        # way and will throw a ValueError
+        pass
```

Comment:
Can we add an issue to handle this?
Currently, the fields will be empty which is not ideal.

Reply:
filed https://github.com/groq/mlagility/issues/318

Reply:
https://github.com/groq/mlagility/pull/317/commits/de2f36182ad8bfffa05d67aa0c3ea605706da6ba